check_version_hook.sh

```diff
@@ -0,0 +1,9 @@
+# Check for version alignment between RN and Android SDK
+rn_version=$(awk '/"version"/ {print $2;exit}' package.json | tr -d '",')
```

Comment:
It's possible to omit the `tr(1)` command and be more specific about which version is used.
Note that this requires that the `version` field in the package.json is on the line after the name field.
Might be problematic…
```suggestion
rn_version=$(awk -F'"' '/name.*react-native-plaid-link-sdk/ {getline; print $4}' package.json)
```